### PR TITLE
Add/is this the weekend

### PR DESCRIPTION
**Pull request by youknowriad**
Adding a `is it the weekend` question to the bot. Something like http://estcequecestbientotleweekend.fr/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,34 @@
-# DZ Community Bot 🤖
```
Comment (youknowriad) [inline README.md]: Let me rebase

**Comment by youknowriad** [comment]
oops lint error, it should be a precommit hook :)

**Comment by IOAyman** [comment]
@youknowriad already working on it ;)

**Review by IOAyman — APPROVED** [review]

**Comment by youknowriad** [comment]
ship it 🎉

**Comment by IOAyman** [comment]
Can you implement a the same functionality but in a reminder fashion? For instance, on Wednesday it reminds you to plan something cool for the weekend?

**Comment by youknowriad** [comment]
@IOAyman Yes, I thought about it, it will be awesome, I'll do it later ;)

**Comment by IOAyman** [comment]
Great 👍

**Pull request merged**